fastlane_core/lib/fastlane_core/helper.rb

```diff
@@ -66,6 +66,11 @@ def self.iterm?
       !!ENV["ITERM_SESSION_ID"]
     end
 
+    # Logs base directory
+    def self.buildlog_path
+      return ENV["FL_BUILDLOG_PATH"] || "~/Library/Logs"
```

Comment:
You mean for both values or just the default with the `~`? I left then unexpanded because they were in the places I call this from, the option defaults are unexpanded paths. For example gym prints a table of the options it's using and that table reads "~/Library/Logs/gym" if I expand it now it will be the complete path?